debian/neo-go.postinst

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+# postinst script for neo-go
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    configure)
+        USERNAME=neo-go
+        id -u $USERNAME >/dev/null 2>&1 || useradd -s /usr/sbin/nologin -d /var/lib/$USERNAME --system -M -U -c "Neo-go consensus node" neo-go
```

Comment:
Nitpicking, but it's not always the consensus node. E.g. it can be just an RPC node.

Reply:
Service file is installed in inactive and stopped state, so it should not break anything.
Probably, having different unit files for different operation modes could be usefull?
If so I'll add them.

Reply:
It doesn't need additional unit files, consensus/RPC/any other service operation only depends on configuration. But the comment for the user is incorrect, is should be something like "NeoGo node" and that's it.